vllm/_custom_ops.py

```diff
@@ -153,6 +153,36 @@ def marlin_gemm(a: torch.Tensor, b_q_weight: torch.Tensor,
                                 size_n, size_k)
 
 
+# cutlass
+def cutlass_scaled_mm_dq(
+        a: torch.Tensor,
+        b: torch.Tensor,
+        a_scales: torch.Tensor,
+        b_scales: torch.Tensor,
+        out_dtype: Type[torch.dtype] = torch.bfloat16) -> torch.Tensor:
```

Comment:
Shall we remove the default argument here? It is not clear that `torch.bfloat16` is more canonical than torch.float16 and can lead to bugs if the caller doesn't specify it -- better to be explicit here :)